scripts/screeps_rl_simulator_harness.py

```
@@ -510,6 +516,426 @@ def _safe_filename(value: str) -> str:
+    command = [*compose, "exec", "-T", "mongo", "mongosh", "--quiet", "--eval", eval_script]
+    result = smoke.run_command(command, cfg, timeout=60, output_limit=200000)
+    if result.get("returncode") != 0:
+        return None
+    try:
+        payload = json.loads(str(result.get("output_excerpt", "")).strip().splitlines()[-1])
+    except (IndexError, json.JSONDecodeError):
+        return None
+    return find_runtime_parameter_consumption_evidence(payload)
+
+
+def find_runtime_parameter_consumption_evidence(payload: Any) -> JsonObject | None:
```

Ruff (0.15.13)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `payload`

(ANN401)